addons/mail/static/src/core/common/message_seen_indicator_popover.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<templates xml:space="preserve">
+
+    <t t-name="mail.MessageSeenIndicatorPopover" owl="1">
+        <div class="o-mail-MessageSeenIndicatorPopover popover" role="tooltip">
+            <div class="popover-body">
+                <ul class="list-unstyled m-0">
+                    <li t-foreach="props.message.channelMemberHaveSeen.slice(0, 10)" t-as="member" t-key="member.id">
+                        <t t-call="mail.MessageSeenIndicatorPopover.card" />
+                    </li>
+                </ul>
+                <small t-if="props.message.channelMemberHaveSeen.length > 10">
+                    and <a role="button" t-on-click="openModal"><t t-out="props.message.channelMemberHaveSeen.length - 10"/> others</a>
+                </small>
```

Comment:
Not good for translation, because they are split in 2 terms `_t("and ")` + `_t(" others")` and we can't move the placement of number.

Do do partial style, you need to follow a similar pattern as `CANCEL_OR_SAVE_EDIT_TEXT` in `Composer` component